web/src/services/api/sceneApi.ts

```diff
@@ -23,6 +23,7 @@ export type SceneExtension = {
 export type SceneQueryProps = {
   sceneId?: string;
   lang?: string;
+  pollInterval?: number;
```

Comment:
Is this being used?

Reply:
Its not, https://www.apollographql.com/docs/react/data/queries/#polling

Reply:
Ahh sorry yeah can you for now remove the poll. You can leave the typings as is, just remove where you are passing the actual poll value. 
The reason is because I am worried about performance issues around using polling, and want to investigate just how far apollo client's polling gets us with "real-time" working together on the same project.
If it is a good solution, I want to setup polling on most queries. But also want to investigate other ways to keep synced up.

Reply:
Done.